### PR TITLE
Cookbooks folder conversion: gplates

### PR DESCRIPTION
**Pull request by tjhei**
part of #3422 (moving cookbooks into folders and split manual source files)
also rename files to use underscores as proposed in #3255

### CODE REVIEW

File: cookbooks/gplates/doc/gplates.tex [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+observables in order to gain information about the initially assumed geologic
+input. In this cookbook we present \aspect{}'s interface to the  widely used
+plate reconstruction software GPlates, and the steps to go from a geologic plate
+reconstruction to a geodynamic model incorporating these velocities as boundary
+condition.
+
+\paragraph{Acquiring a plate reconstruction.}
+
+The plate reconstruction that is used in this cookbook is included in
+the \texttt{data/velocity-boundary-conditions/gplates/} directory of
```
Comment (gassmoeller) [inline cookbooks/gplates/doc/gplates.tex:18]:
```suggestion
the \texttt{data/boundary-velocity/gplates/} directory of
```

File: cookbooks/gplates/doc/gplates.tex [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+between pairs of files to the current model time. To achieve this, one needs
+to use the \texttt{\%d} wildcard in the velocity file name, which represents the current
+velocity file index (e.g. \texttt{time\_dependent.\%d.gpml}). This index is
+calculated by dividing the current model time by the user-defined time step
+between velocity files (see parameter file above). As the model time progresses
+the plugin will update the interpolation accordingly and if necessary read in
+new velocity files. In case it can not read the next velocity file, it assumes
+the last velocity file to be the constant boundary condition until the end of
+the model run. One can test this behavior with the provided data files
+\texttt{data/velocity\_boundary\_conditions/gplates/time\_dependent.\%d.gpml}
```
Comment (gassmoeller) [inline cookbooks/gplates/doc/gplates.tex:199]:
```suggestion
\texttt{data/boundary\_velocity/gplates/time\_dependent.\%d.gpml}
```

Comment (tjhei) [reply cookbooks/gplates/doc/gplates.tex]: it is actually ``data/boundary-velocity/``...

**Comment by tjhei** [comment]
fixed.
